impact.tex

```diff
@@ -57,7 +54,26 @@ \subsection{Pathway toward impact}
 \end{draft}
 
 
-\TOWRITE{}{Start with brief narrative summary, hitting 'Scale and Significance' hard. We have close to nothing on Significance, yet. Scale could be clearer: All researchers!}
+Almost all researchers perform some kind of computation in the course of their work,
+from production of simple charts and figures to large-scale simulations.
+Researchers today are faced with a great variety of useful
+and increasingly open tools to perform their research tasks.
+However, this variety presents its own challenge.
+Because we are building and extending \textbf{generic tools for computational reproducibility},
+we help researchers of all kinds perform their work reproducibly without dictating
+how they do the research part of their work.
+By focusing on (\textbf{reproducible computational environments}),
+and offering a generic and interoperable solution,
+we \emph{are} solving a big piece of the reproducibility problem
+and act as a bridge between any solutions to other parts of the reproducibility problem,
+for instance those classically addressed by workflow management systems.
+"Do one thing and do it well" is key to success and will accelerate progress
+by allowing others to concentrate on other challenges of the reproducibility problem.
```

Comment:
I don't know if using the word "different" would make it any better (we repeat 'other' twice here):

by allowing others to concentrate on different challenges of the reproducibility problem.